packages/opds-web-client/src/__tests__/authMiddleware-test.ts

```diff
@@ -36,10 +20,11 @@ describe("authMiddleware", () => {
   beforeEach(() => {
     dataFetcher = new DataFetcher();
     dataFetcher.clearAuthCredentials();
-    hideAuthFormStub = stub();
-    clearAuthCredentialsStub = stub();
-    showAuthFormStub = stub();
-    actionCreatorStub = stub(ActionCreator, "default", MockActionCreator);
+    // clearAuthCredentialsStub = stub();
+    // showAuthFormStub = stub();
```

Comment:
you can remove these.

Reply:
This is weird. I did remove them and they're not showing up anymore locally or in this file but it is for this PR change view.